clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/CreateServerGroupAtomicOperation.java

```diff
@@ -507,12 +503,44 @@ protected CreateServiceRequest makeServiceRequest(
             .withServiceRegistries(serviceRegistries)
             .withDeploymentConfiguration(deploymentConfiguration);
 
+    List<Tag> taskDefTags = new LinkedList<>();
```

Comment:
nit: can you make this a set to avoid adding duplicate tags

Reply:
The AWS API expects a list of `Tag` objects so changing this to a set increases the complexity. The `CreateServerGroupDescription.tags` field is already a `Map<String, String>` so it's not possible to pass in tags with duplicate keys. I've added a test to verify this.

Reply:
It's fairly straightforward in that you could do `Set<String> hSet = new HashSet<String>(taskDefTags);`, but that being said, I don't see the need at the moment.